manager/src/main/docker/Dockerfile.jvm

```diff
@@ -1,55 +0,0 @@
-####
```

Comment:
This was an initial leftover, we have a `docker` folder in the root for the dockerfiles of the applications